fastlane/lib/fastlane/actions/install_provisioning_profile.rb

```diff
@@ -0,0 +1,58 @@
+require 'shellwords'
+
+module Fastlane
+  module Actions
+    class InstallProvisioningProfileAction < Action
+      def self.run(params)
+        absolute_path = File.expand_path(params[:provisioning_profile_path])
+        FastlaneCore::ProvisioningProfile.install(absolute_path)
+      end
+
+      def self.description
+        "Install provisioning profile from inputfile"
+      end
+
+      def self.details
+        "Install provisioning profile from inputfile for current user"
+      end
+
+      def self.authors
+        ["SofteqDG"]
+      end
+
+      def self.category
+        :code_signing
+      end
+
+      def self.is_supported?(platform)
+        [:ios, :mac].include?(platform)
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :provisioning_profile_path,
+                                  env_name: "FL_INSTALL_PROVISIONING_PROFILE_PATH",
+                               description: "Path to provisioning profile",
+                                  optional: false,
+                                      type: String,
+                              verify_block: proc do |value|
+                                absolute_path = File.expand_path(value)
+                                unless File.exist?(absolute_path)
+                                  UI.user_error!("Failed installation of provisioning profile from file at path: '#{value}'")
```

Comment:
This should probably just say Could not find provisioning profile at path: ..., as it doesn't try to install this yet.

Reply:
Message changed